rmw/include/rmw/service_endpoint_info_array.h

```diff
@@ -0,0 +1,157 @@
+// Copyright 2025 Minju Lee (이민주). All rights reserved.
```

Comment:
we can attribute ourselves, but "all right reserved" is extra. see appendix on https://opensource.org/license/apache-2-0

```suggestion
// Copyright 2025 Minju Lee (이민주).
```